src/controllers/controller.cpp

```diff
@@ -108,14 +108,19 @@ void Controller::receive(const QByteArray data) {
     int length = data.size();
     if (debugging()) {
         // Formatted packet display
-        QString message = QString("%1: %2 bytes:\n").arg(m_sDeviceName).arg(length);
-        for(int i=0; i<length; i++) {
-            QString spacer=" ";
-            if ((i+1) % 4 == 0) spacer="  ";
-            if ((i+1) % 16 == 0) spacer="\n";
-            message += QString("%1%2")
-                        .arg((unsigned char)(data.at(i)), 2, 16, QChar('0')).toUpper()
-                        .arg(spacer);
+        QString message =
+                QString("%1: %2 bytes:\n").arg(m_sDeviceName).arg(length);
+        for (int i = 0; i < length; i++) {
+            QString spacer = " ";
+            if ((i + 1) % 4 == 0)
+                spacer = "  ";
+            if ((i + 1) % 16 == 0)
+                spacer = "\n";
+            message +=
+                    QString("%1%2")
+                            .arg((unsigned char)(data.at(i)), 2, 16, QChar('0'))
+                            .toUpper()
+                            .arg(spacer);
```

Comment:
Stuff like this makes me want 2-space indent too, though I normally prefer 4. :)